Jenkinsfile

```diff
@@ -1083,7 +1084,7 @@ pipeline {
                         }
                     }
                 } // stage('Test CentOS 7 RPMs')
-                stage('Test RPMs on Leap 15.4') {
+                stage('Test RPMs on Leap 15.5') {
```

Comment:
Ditto for this stage.  This one is more informational than the EL 8.6 stage as support for 15.4 is ending soon but it's still useful for us to know and to be able to tell customers who may not be able to switch as quickly as SUSE would like them to that our basic sanity testing still passes on 15.4.